include/termcolor/termcolor.hpp

```diff
@@ -383,6 +383,134 @@ namespace termcolor
     }
 
 
+    inline
+    std::ostream& high_grey(std::ostream& stream)
+    {
+        if (_internal::is_colorized(stream))
+        {
+            #if defined(TERMCOLOR_OS_MACOS) || defined(TERMCOLOR_OS_LINUX)
+            stream << "\033[90m";
```

Comment:
nit: Here and below, could you please the same indentation policy that has been esablished for manipulators?